[2022-02-02 21:39:49,900] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.halo manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:49,915] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.halo manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:49,916] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:49,916] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 21:39:49,916] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:49,932] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): halo> on 2022-02-02 21:39:36.511344+00:00
[2022-02-02 21:39:49,936] {standard_task_runner.py:52} INFO - Started process 2016 to run task
[2022-02-02 21:39:49,939] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question4_ZONE', 'halo', 'manual__2022-02-02T21:39:36.511344+00:00', '--job-id', '367', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions4.py', '--cfg-path', '/tmp/tmpxoehxgm9', '--error-file', '/tmp/tmpdrl5s_3p']
[2022-02-02 21:39:49,947] {standard_task_runner.py:77} INFO - Job 367: Subtask halo
[2022-02-02 21:39:50,021] {logging_mixin.py:109} INFO - Running <TaskInstance: Question4_ZONE.halo manual__2022-02-02T21:39:36.511344+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:39:50,095] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question4_ZONE
AIRFLOW_CTX_TASK_ID=halo
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T21:39:36.511344+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-02T21:39:36.511344+00:00
[2022-02-02 21:39:50,096] {logging_mixin.py:109} INFO - hello world!!
[2022-02-02 21:39:50,096] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:39:50,109] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question4_ZONE, task_id=halo, execution_date=20220202T213936, start_date=20220202T213949, end_date=20220202T213950
[2022-02-02 21:39:50,131] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:39:50,168] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
